HAL/keymaster/4.1/JavacardKeymaster4Device.cpp

```diff
@@ -873,8 +1074,7 @@ Return<void> JavacardKeymaster4Device::begin(KeyPurpose purpose, const hidl_vec<
             cborConverter_.getKeyParameters(item, 1, outParams);
```

Comment:
For all the cborConverter calls we are not capturing the return values. Check them.

Reply:
Done.